src/test/java/com/puppycrawl/tools/checkstyle/DefaultLoggerTest.java

```diff
@@ -166,4 +199,114 @@ public void testFinishLocalSetup() {
+        message.setAccessible(true);
+        final Map<String, ResourceBundle> bundleCache =
+                Whitebox.getInternalState(message.getDeclaringClass(), "BUNDLE_CACHE");
+        assertEquals("Une erreur est survenue {0}", message.invoke(messageClass),
+                "Invalid message");
+        assertEquals(1, bundleCache.size(), "Invalid bundle cache size");
+    }
+
+    @Test
+    public void testNullArgs() throws Exception {
+        final Constructor<?> cons = getConstructor();
+        cons.setAccessible(true);
+        final Object messageClass = cons.newInstance(DefaultLogger.ADD_EXCEPTION_MESSAGE,
+                new String[] {"myfile"});
+        final Method message = messageClass.getClass().getDeclaredMethod("getMessage");
+        message.setAccessible(true);
+        final String output = (String) message.invoke(messageClass);
+        assertTrue(output.contains("Error auditing myfile"),
+                "Violation should contain exception info, but was " + output);
+
+        final Object nullClass = cons.newInstance(DefaultLogger.ADD_EXCEPTION_MESSAGE, null);
+        final Method nullMessage = nullClass.getClass().getDeclaredMethod("getMessage");
+        nullMessage.setAccessible(true);
+        final String outputForNullArgs = (String) nullMessage.invoke(nullClass);
+        assertTrue(outputForNullArgs.contains("Error auditing {0}"),
+                "Violation should contain exception info, but was " + outputForNullArgs);
+    }
+
+    private static Constructor<?> getConstructor() throws Exception {
+        final Class<?> message = getDefaultLoggerClass().getDeclaredClasses()[0];
+        return message.getDeclaredConstructor(String.class, String[].class);
+    }
+
+    private static Object getAuditStartMessageClass() throws Exception {
+        final Constructor<?> cons = getConstructor();
+        return cons.newInstance("DefaultLogger.auditStarted", null);
+    }
+
+    private static Object getAuditFinishMessageClass() throws Exception {
+        final Constructor<?> cons = getConstructor();
+        return cons.newInstance("DefaultLogger.auditFinished", null);
+    }
+
+    private static Method getAuditStartMessage() throws Exception {
+        final Constructor<?> cons = getConstructor();
+        cons.setAccessible(true);
+        final Object auditStartMessageClass = getAuditStartMessageClass();
+        return auditStartMessageClass.getClass().getDeclaredMethod("getMessage");
+    }
+
+    private static Method getAuditFinishMessage() throws Exception {
+        final Constructor<?> cons = getConstructor();
+        cons.setAccessible(true);
+        final Object auditFinishMessageClass = getAuditFinishMessageClass();
+        return auditFinishMessageClass.getClass().getDeclaredMethod("getMessage");
+    }
+
+    private static Class<?> getDefaultLoggerClass() throws Exception {
+        return Class.forName("com.puppycrawl.tools.checkstyle.DefaultLogger");
```

Comment:
com.puppycrawl.tools.checkstyle.DefaultLogger.class

Reply:
This change cannot be done as class is not found by reflection

`java.lang.ClassNotFoundException: com.puppycrawl.tools.checkstyle.DefaultLogger.class`